MapboxNavigation/Constants.swift

```diff
@@ -15,6 +15,8 @@ public let RouteControllerProgressDidChange = Notification.Name(MBRouteControlle
 public let RouteControllerAlertLevelDidChange = Notification.Name(MBRouteControllerAlertLevelDidChange)
 public let RouteControllerShouldReroute = Notification.Name(MBRouteControllerShouldReroute)
 
+public let RouteControllerDidReroute = Notification.Name(MBRouteControllerDidReroute)
```

Comment:
RouteController or MapboxNavigation.swift never use this notification. Should we add a Constants.swift to MapboxNavigationUI.swift which is responsible for firing the notification and put it there instead?